ui/packages/catalog-admin-module-sources/src/main/webapp/js/model/Status.js

```diff
@@ -17,13 +17,9 @@ define(['backbone'], function(Backbone) {
   var Status = {}
 
   Status.Model = Backbone.Model.extend({
-    url:
+    urlRoot:
       '../jolokia/exec/org.codice.ddf.catalog.admin.poller.AdminPollerServiceBean:service=admin-source-poller-service/sourceStatus/',
     model: Status.Model,
-
-    initialize: function(pid) {
```

Comment:
Just curious, what does removing this do?

Reply:
I needed a way to update the pid on the model. Using [urlRoot](http://backbonejs.org/#Model-urlRoot) allows this and was cleaner than refactoring to add a `setPid` method.